Kickstarter-iOS/Views/Controllers/SettingsNotificationsViewController.swift

```diff
@@ -21,7 +25,11 @@ internal final class SettingsNotificationsViewController: UIViewController {
     tableView.dataSource = dataSource
     tableView.delegate = self
 
+    emailFrequencyPickerView.delegate = self
```

Comment:
Think we should prefix all of these property references with `self`

Reply:
Just curious, how come? I thought this wasn't necessary unless you're referencing `self` in a closure

Reply:
Unless it's just for consistency since we do it everywhere else in the repo?

Reply:
Oh yeah, if for no other reason it'd be for consistency with the rest of the codebase. I personally think it's handy to know if something is global or specific to the instance just by reading it but I know it's also something of personal preference.